muxplore/ui/ui_scrExplore.c

```diff
@@ -45,9 +45,9 @@ void ui_scrExplore_screen_init(void)
     lv_obj_set_height(ui_pnlContent, 392);
     lv_obj_set_x(ui_pnlContent, 0);
     lv_obj_set_y(ui_pnlContent, 44);
-    lv_obj_set_align(ui_pnlContent, LV_ALIGN_TOP_MID);
```

Comment:
Related to discord thread:
https://discord.com/channels/1152022492001603615/1262404136863928475/1262404136863928475

With this set align to center if you tried to change theme->MISC.CONTENT.WIDTH the content would end up centered with no way to adjust alignment.  With this updated to left align you have full control over the alignment using theme->MISC.CONTENT.PADDING_LEFT.  This was also causing an issue where when using theme->MISC.CONTENT.PADDING_LEFT you would have to double the value in order to get the right padding.

![image](https://github.com/user-attachments/assets/0ab497f6-9687-421f-9c97-d7fed72ebb00)
![image](https://github.com/user-attachments/assets/8df1cd19-26ad-41a2-a46f-104421066a19)
![image](https://github.com/user-attachments/assets/1427f343-06e0-41cd-a49c-3f8d1eccf2b6)